go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/jarcoal/httpmock v1.0.4
 	github.com/mattermost/mattermost-plugin-api v0.0.9
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200313113657-e2883bfe5f37
+	github.com/mattermost/mattermost-server/v5 v5.24.0
```

Comment:
Could some commit on `master` be used instead?

Reply:
This was discussed on a different conversation. We will point to master, merge this, and whenever 5.24 is addressable, we will update the plugin to address the correct server version.